drivers/sensors/lepton.c

```diff
@@ -369,11 +367,8 @@ static int reset(omv_csi_t *csi) {
         return OMV_CSI_ERROR_CTL_FAILED;
     }
 
-    if (vospi_initialized == false) {
-        if (vospi_init(lepton.v_res, _vospi_buf) != 0) {
-            return OMV_CSI_ERROR_CTL_FAILED;
-        }
-        vospi_initialized = true;
```

Comment:
Why initialize the bus on every reset? Also, we need to check fb!=NULL because it's only allocated when `csi` object is created.

Reply:
Done